sdk/tables/Azure.Data.Tables/src/swagger/TableRestClient.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Data.Tables
+{
+    internal partial class TableRestClient
+    {
+        internal HttpMessage CreateDeleteRequest(string table, string requestId)
+        {
+            var message = _pipeline.CreateMessage();
+            var request = message.Request;
+            request.Method = RequestMethod.Delete;
+            var uri = new RawRequestUriBuilder();
+            uri.AppendRaw(url, false);
+            uri.AppendPath("/Tables('", false);
+            uri.AppendPath(table, true);
+            uri.AppendPath("')", false);
+            request.Uri = uri;
+            request.Headers.Add("x-ms-version", version);
+            if (requestId != null)
+            {
+                request.Headers.Add("x-ms-client-request-id", requestId);
+            }
+            // Delete requests fail without this header.
```

Comment:
Should we add it to swagger?

Reply:
It's debatable. I think the answer is yes.

Comparing notes with what happens in python, this header appears to not be required, although is harmless if added. In addition, I was able to run fiddler against Azure Storage Explorer in the portal, and it sends this header. The [documentation](https://docs.microsoft.com/en-us/rest/api/storageservices/delete-table#request-headers) doesn't mention it.

Reply:
Let's comment on the swagger PR at least and see how that goes.